src/lib/djerba/plugins/fusion/plugin.py

```diff
@@ -6,19 +6,25 @@
 import logging
 import os
 import re
+import glob
+import warnings
```

Comment:
No need to import and use `warnings` when we have a built in logger. The logger has benefits like putting a timestamp on the warning, and optionally writing it to a logfile.